Dockerfile.template

```diff
@@ -6,7 +6,7 @@ LABEL maintainer="Phase2 <outrigger@phase2technology.com>" \
   org.label-schema.vendor="Phase2 <outrigger@phase2technology.com>" \
   org.label-schema.name="Node v%%NODE_VERSION%%" \
   org.label-schema.vcs-url="https://github.com/phase2/docker-node" \
-  org.label-schema.docker.cmd="docker run --rm -it outrigger/node:8" \
+  org.label-schema.docker.cmd="docker run --rm -it outrigger/node:%%NODE_VERSION%%" \
```

Comment:
Is it possible to provide a default value for this? Maybe something similar to:
```
# Allow overrides to the build environment.
ARG BUILD_ENVIRONMENT=production
ENV NODE_ENV=$BUILD_ENVIRONMENT
```

Reply:
Nevermind.  The NODE_VERSION is substituted by template.sh based on the directory we're building.